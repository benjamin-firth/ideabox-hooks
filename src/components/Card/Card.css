.card {
  background-color: #fff;
  border: 2px solid #008080;
  height: 200px;
  margin: 20px;
  padding: 10px;
  width: 200px;
}