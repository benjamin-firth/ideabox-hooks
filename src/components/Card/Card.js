import React from 'react';
import './Card.css';

const Card = ({ id, name, date, time, number, deleteReservation}) => {

  return(
    <div className="card">
      <h3>{name}</h3>
      <p>{date}</p>
      <p>{time} pm</p>
      <p>Number of Guests: {number}</p>
      <button type="button" id={id} onClick={() => deleteReservation(id)}>Cancel</button>
    </div>
  )
}

export default Card;