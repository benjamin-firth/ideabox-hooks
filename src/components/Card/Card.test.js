import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

describe('Card', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Card />)
  })

  describe('Card Snapshots', () => {
    it('Should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    })
  })

})