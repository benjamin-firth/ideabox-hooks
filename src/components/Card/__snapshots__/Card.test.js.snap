// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Card Snapshots Should match the snapshot 1`] = `
<div
  className="card"
>
  <h3 />
  <p />
  <p>
     pm
  </p>
  <p>
    Number of Guests: 
  </p>
  <button
    onClick={[Function]}
    type="button"
  >
    Cancel
  </button>
</div>
`;
